fimsauthz-api/fims-authz.yaml

```diff
@@ -212,11 +230,12 @@ components:
           description: see definitions/JwsCompactSerialization in this spec
         scope:
           type: string
-          description:  The granted scope(s), a list of space-delimited,
-          case-sensitive names. Same as in RFC 6749.
+          description:  |
+            The granted scope(s), a list of space-delimited,
+            case-sensitive names. Same as in RFC 6749.
 
-          Version 1.0.2 of this specificaiton allows a pair of scope for this case only:
-          [dss.read.identification_service_areas, rid.read.enhanced_details]
+            Version 1.0.2 of this specificaiton allows a pair of scope for this case only:
```

Comment:
Version is now changed as per this PR.  Change all of these notes; better to find a generic way perhaps "For NASA 2020 events".

Reply:
will take a look... thanks.

Reply:
done i think.  i just put "Since 1.0.2..." for example.